appveyor.yml

```diff
@@ -61,26 +61,47 @@ for:
 
       # Install AWS CLI Globally outside of a venv.
       - "pip install awscli"
+      
```

Comment:
I think an explanation of what envs will be created, what is the motivation, what each env is for here is good for people without context.

Reply:
Added more comments